packages/pong-engine/src/game.ts

```diff
@@ -19,14 +28,26 @@ export const paddleMoveRight = (state: GameState): GameState => {
   return reducer(state, { type: 'paddleMoveRight', payload: {} });
 };
 
+export const paddleOpponentMoveRight = (state: GameState): GameState => {
```

Comment:
I think this concept of naming these commands `xxxOpponentXxx` and `xxxXxx` is not a really good idea. When we have two users interacting, which is the main player and which is the opponent? Why not just player A and player B?

Reply:
The opponent is just a point of view, when you are the player the opponent is the other and viceversa. If we name them as playerA and playerB that would be harder to understand since it does not have any semantic meaning